KaylaKornelis-app/src/main/java/edu/KaylaKornelis/advancedjava/Assignment3/StockQuoteApplication.java

```diff
@@ -0,0 +1,60 @@
+package edu.KaylaKornelis.advancedjava.Assignment3;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * This class is used to retrieve stock quotes
+ */
+public class StockQuoteApplication {
+    
+    public static void main(String[] args){
+        
+        System.out.println(createListOfStockquotes("APPL", "1/1/2014", "2/1/2014"));
+        
+        System.out.println(createBasicStock("APPL"));
+  
+    }
+    
+        public static List<StockQuote> createListOfStockquotes(String symbol, String from, String until){
+            SimpleDateFormat sdf = new SimpleDateFormat("MM/dd/YYYY");
+            Calendar fromDate = Calendar.getInstance();
+            
+            try{
+                fromDate.setTime(sdf.parse(from));
+            }catch(ParseException e){
+                e.printStackTrace();
```

Comment:
Never just catch an exception and print the stack trace. handle the exception instead. Please review my notes on exception handling as well as chapter in Bloch. Let me know if you have any questions.

Reply:
Added exception handling.